gtk4/src/auto/print_unix_dialog.rs

```diff
@@ -223,6 +221,16 @@ impl PrintUnixDialog {
         }
     }
 
+    #[doc(alias = "print-settings")]
```

Comment:
those are repeated with get_settings?

Reply:
Fixed. It was missing annotations in gtk, fixed there.